src/components/chat-view-container/chat-view.tsx

```diff
@@ -274,6 +274,8 @@ export class ChatView extends React.Component<Properties, State> {
             onClose={this.closeLightBox}
           />
         )}
+        {/* See: ZOS-115
+         * @ts-ignore */}
```

Comment:
I'm a little confused here. I thought adding children to the pros was supposed to remove the need for the type ignore?

Reply:
Yeah you're right - I left those there by accident. Thanks for catching that!